### PR TITLE
Add sphinx theme 1.0

### PR DESCRIPTION
**Pull request by dgarcia360**
Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/133

Sphinx ScyllaDB Theme 1.0 is now released 🥳 

We’ve made a number of updates to the look and feel of the theme to improve the overall user experience.
You can read more about all notable changes [here](https://sphinx-theme.scylladb.com/stable/CHANGELOG#september-2021).

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. You will see the docs with the new look and feel:

image

## Before you merge this PR...

I've proposed a new landing page for the project (index.rst). Please, review it and let me know if you would like to change the distribution of the cards, change the text, or use different icons/images.

### CODE REVIEW

**Comment by dgarcia360** [comment]
@tzach @lauranovich Please review

**Comment by tzach** [comment]
I was expecting only the theme version to change. I now understand the upgrade have many more updates with it.
(not a problem)

File: .github/workflows/docs-pages@v2.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "Publish docs to GitHub Pages"
```
Comment (tzach) [inline .github/workflows/docs-pages@v2.yaml:1]: @dgarcia360 can you please add a doc of this yml?
If we will have this in all projects, better put it in the theme wiki

**Comment by tzach** [comment]
The Scylla link is missing in the top right (near the DOCUMENTION)
image

**Comment by tzach** [comment]
The specific project top menu are gone.
Not all proejcts need the same top menu options

### New
image

### old
image

**Comment by lauranovich** [comment]
https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally - in case the link for checkout out PRs locally is delivered in Spanish - here is the English version

**Comment by lauranovich** [comment]
> The Scylla link is missing in the top right (near the DOCUMENTION)
> image

Tzach check your cache - I see the icon.

**Comment by lauranovich** [comment]
This is what I see

Screenshot from 2021-09-27 09-50-14

**Comment by lauranovich** [comment]
I think it would have been better if the change to the project Home Page was done as a standalone PR
this way it would be easier to see what is in that specific change vs the theme update.

File: .github/workflows/docs-pages@v2.yaml (hunk 1 above)
Comment (dgarcia360) [reply .github/workflows/docs-pages@v2.yaml:1]: See https://sphinx-theme.scylladb.com/stable/github-pages#available-workflows

**Comment by tzach** [comment]
Thanks @lauranovich it was a cache issue, but it likely all of our users will hit it

**Comment by dgarcia360** [comment]
> The specific project top menu are gone.

We talked about having a shared theme links between projects. This makes navigating between documentation projects consistent. If you prefer to have different links per projects, please open a new issue on sphinx-scylladb-theme.

> The Scylla link is missing in the top right (near the DOCUMENTION)

For the next version of the theme, I can change the name of the file (from `logo.png` to `scylla-logo.png`). This change will  force browsers to download the latest version of the icon.

> I think it would have been better if the change to the project Home Page was done as a standalone PR
this way it would be easier to see what is in that specific change vs the theme update.

No problem, I'm moving the new page to a separate PR.

**Comment by dgarcia360** [comment]
I removed the custom `index.rst` file.
If you prefer to add it back once you merge this PR:

1. Copy and paste the contents from [this gist](https://gist.githubusercontent.com/dgarcia360/8aed0be5822e1ceed1439d55881163ba/raw/cb2fa9967b097c5b34489cf0a413c3e44498ea59/index.rst
) into `docs/index.rst`.

2. In the theme's ``conf.py``, set `html_theme_options.hide_sidebar_index` to ``'true'``. Example:

```
html_theme_options = {
    'hide_sidebar_index': 'true',
}
```

**Comment by lauranovich** [comment]
Created https://github.com/scylladb/care-pet/issues/36 to follow up

**Comment by tzach** [comment]
> The specific project top menu are gone.
> 
> We talked about having a shared theme links between projects. This makes navigating between documentation projects consistent. If you prefer to have different links per projects, please open a new issue on sphinx-scylladb-theme.

I was not aware.
We must have a quick way to go back to the *project* HP, like we have here (for monitoring)
image

**Comment by lauranovich** [comment]
Can we have a Home icon in the sidebar and always have the sidebar present?  would this work?

**Comment by tzach** [comment]
> Can we have a Home icon in the sidebar and always have the sidebar present? would this work? 

I do not know, need to see an example

**Comment by dgarcia360** [comment]
Now we have this link in the breadcrumbs:

image


In a subequent release, we could make this link more visible. For example:

image


What do you think?

**Comment by tzach** [comment]
> What do you think?

Great!

**Comment by lauranovich** [comment]
I like it too

**Comment by lauranovich** [comment]
@dgarcia360 - all set?

**Comment by tzach** [comment]
The problem with the Home solution, is there no way to know in which doc project you are, except from the URL.
In this page for example, nothing says care-pet!

I think we still need a customize project name somewhere on the top menu

image

**Pull request merged**